package.json

```diff
@@ -4,7 +4,8 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "dev": "export NODE_ENV=development && nodemon src/temp-server.js ",
+    "test": "export NODE_ENV=testing && nodemon src/temp-server.js "
```

Comment:
You need not export NODE_ENV on develop. Develop is picked up automatically

Reply:
@julietezekwe I know that Sequelize by default `NODE_ENV=develop`. Now, It is working let me remove it.